.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,169 @@
+name: Run Cookietemple Tests
+
+on:
+    - push
+    - pull_request
+
+jobs:
+    tests:
+        name: ${{ matrix.session }} ${{ matrix.python-version }} / ${{ matrix.os }}
+        runs-on: ${{ matrix.os }}
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - {
+                          python-version: 3.8,
+                          os: ubuntu-latest,
+                          session: "pre-commit",
+                      }
+                    - {
+                          python-version: 3.8,
+                          os: ubuntu-latest,
+                          session: "safety",
+                      }
+                    - {
+                          python-version: 3.8,
+                          os: ubuntu-latest,
+                          session: "mypy",
+                      }
+                    - {
+                          python-version: 3.8,
+                          os: ubuntu-latest,
+                          session: "tests",
+                      }
+                    - {
+                          python-version: 3.8,
+                          os: windows-latest,
+                          session: "tests",
+                      }
+                    - {
+                          python-version: 3.8,
+                          os: macos-latest,
+                          session: "tests",
+                      }
+                    - {
+                          python-version: 3.8,
+                          os: ubuntu-latest,
+                          session: "typeguard",
+                      }
+                    - {
+                          python-version: 3.8,
+                          os: ubuntu-latest,
+                          session: "xdoctest",
+                      }
+                    - {
+                          python-version: 3.8,
+                          os: ubuntu-latest,
+                          session: "docs-build",
+                      }
+
+        env:
+            NOXSESSION: ${{ matrix.session }}
+
+        steps:
+            - name: Check out the repository
+              uses: actions/checkout@v2.3.4
+
+            - name: Set up Python ${{ matrix.python-version }}
+              uses: actions/setup-python@v2.1.4
+              with:
+                  python-version: ${{ matrix.python-version }}
+
+            - name: Install Poetry
+              run: |
+                  pip install poetry
+                  poetry --version
+
+            - name: Install Nox
+              run: |
+                  pip install nox nox-poetry
+                  nox --version
+
+            - name: Install Rich
```

Comment:
I forgot this again. Why are we explicitly installing rich?

Reply:
Rich is used to print the error message in our noxfile if nox or nox-poetry has not been found